maxim/decorators/span.py

```
@@ -117,7 +117,7 @@ async def async_wrapper(*args, **kwargs):
                         evaluator_variables if evaluator_variables is not None else {}
                     )
                 # If actual_trace_id is None, we will try to get the current trace id
-                try:
+                try:                    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)